src/components/Drawer/DrawerExample.tsx

```diff
@@ -24,7 +24,7 @@ export interface Props {
  */
 export const DrawerExample = ({ className, ...other }: Props) => {
   const [drawerOpen, setDrawerOpen] = useState(false);
-  const drawerButton = useRef() as MutableRefObject<HTMLSpanElement>;
```

Comment:
There are 3 changes like this. I don't know why a button ref claimed to be a span or why it's only complaining now, but changing it to a button element makes sense to me 🤷